Sofee.Core.Api/Models/Languages/Exceptions/LanguageValidationException.cs

```diff
@@ -0,0 +1,17 @@
+﻿// -------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE FOR THE WORLD
+// -------------------------------------------------------
+
+using Xeptions;
+
+namespace Sofee.Core.Api.Models.Languages.Exceptions
+{
+    public class LanguageValidationException : Xeption
+    {
+        public LanguageValidationException(Xeption innerException)
+            : base(message: "Language validation error occured. Please, try again.", 
+                       innerException: innerException)
```

Comment:
The parameter aliasing can be skipped here.

Reply:
done